rpcs3/main.cpp

```diff
@@ -85,6 +90,14 @@ int main(int argc, char** argv)
 	qputenv("QT_AUTO_SCREEN_SCALE_FACTOR", "1");
 #endif
 
+#ifdef __linux__
+	struct ::rlimit rlim;
+	rlim.rlim_cur = 4096;
+	rlim.rlim_max = 4096;
+	if (::setrlimit(RLIMIT_NOFILE, &rlim) != 0)
```

Comment:
Is this still necessary after 1193ab0b89da?

Reply:
Yes.